src/api/stripe/stripe-checkout-redirect.ts

```diff
@@ -28,14 +29,14 @@ const stripeCheckoutRedirect = async (
   return await axios
     .post(`${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/api/v1/stripe/subscription`, {
       ...identifier,
+      quantity,
```

Comment:
quantity will function under the hood but will put the purchaser in limbo when they buy team seats currently